blocks/api/raw-handling/test/integration/classic-in.html

```diff
@@ -0,0 +1,4 @@
+<p>First paragraph.</p>
+<p><!--more--></p>
```

Comment:
Would it be worth adding more tests here such as `<p>First <!--more--> paragraph.</p>`?